wallet/src/main/scala/org/bitcoins/wallet/models/OutgoingTransactionTable.scala

```diff
@@ -0,0 +1,79 @@
+package org.bitcoins.wallet.models
+
+import org.bitcoins.core.crypto._
+import org.bitcoins.core.currency._
+import org.bitcoins.core.protocol.transaction._
+import org.bitcoins.core.wallet.fee.SatoshisPerByte
+import slick.jdbc.SQLiteProfile.api._
+import slick.lifted.{PrimaryKey, ProvenShape}
+
+/**
+  * Represents a relevant transaction for the wallet that we should be keeping track of
+  * @param txIdBE Transaction ID
+  * @param txFee fee paid by the transaction
+  * @param expectedFeeRate Fee rate the wallet expected to pay
+  * @param actualFeeRate Fee rate the transaction actually paid
+  */
+case class OutgoingTransactionDb(
+    txIdBE: DoubleSha256DigestBE,
```

Comment:
> Do we want to mention what index our change output is?

I think that is kinda recreating the spending info database, also it could cause problems if there is a tx where we own multiple outcomes (ie coinjoins, payjoins)

Reply:
Otherwise, resolved